src/components/StateForGame.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import { backgroundSpeed } from './../ConstantValues/ConstValues';
+import { Obstacle } from './Game/Obstacle/Obstacle';
+
+export const StateForGame = (): JSX.Element => {
+  const [playerCoord] = useState(100);
+  const [obstacles] = useState([
+    { height: 10, coordX: 1 },
+    { height: 20, coordX: 3 },
+  ]);
```

Comment:
I think that obstacles also require coordY. By the way `x` and `y` do not require `coord` prefix usually.